backend/environment.d.ts

```diff
@@ -14,6 +14,8 @@ declare global {
 			JWT_SIGNUP_SECRET: string;
       MONGO_URL: string;
       NODE_ENV: 'development' | 'staging' | 'testing' | 'production';
+      VERBOSE_ERROR_OUTPUT: boolean;
```

Comment:
VSCode is showing me an error on this one; I think it has to be a `string` type instead because everything passing through as envars enters in `string` format.

<img width="803" alt="Screen Shot 2022-12-23 at 12 20 37 PM" src="https://user-images.githubusercontent.com/25857006/209375717-d5330862-33b8-44b2-acd6-e90072d4c88f.png">

For this reason we have this line in the backend config file for `TELEMETRY_ENABLED`:

`const TELEMETRY_ENABLED = process.env.TELEMETRY_ENABLED! !== 'false' && true;`

Let me know what you think.

Reply:
Interesting for me VSCode showing no error and as far as I now Dotenv will parse it to boolean. But will change for to be sure. 
![image](https://user-images.githubusercontent.com/8071263/209387408-6a02fb52-26fa-4e61-8c0e-82e77f54c4ff.png)